src/app/components/Nav/Nav.scss

```diff
@@ -81,19 +83,19 @@
 #nav .group a {
     display: inline-block;
     padding: 16px;
-    color: var(--gray);
```

Comment:
using this line but extendable comment to all other places where these are swapped. I haven't used sass variables so can't really speak to the benefits/tradeoffs of using sass vars over native css vars. 

I think one reason why I stayed off of it was bc you can't dynamically edit sass vars in js but not sure how important that feature really is.

With that being said, if you think sass vars are worth using, I don't really have much of an opinion.

Reply:
we can use custom properties where dynamic styles are needed, but otherwise, SASS variables provide compile-time guarantees of references, e.g. we are warned when a variable isn't defined